### DISCUSSION TITLE
why font size "fs-5" and "fs-6" are fixed (or not responsive)?

### DISCUSSION THREAD

**Discussion by squashking** [twbs/bootstrap#34717] [General]
I just want the font size to be smaller on smaller screens. But I notice that "fs-6" is always 16px and "fs-5" is always 20px no matter how big the viewport is, but "fs-1" ~"fs-4" seem to be responsive when the viewport is changed.   Aren't they all supposed to be responsive? And it doesn't sound good that the smallest size "fs-6" is 16px which is already very big.

I have tried adding breakpoints to "fs" to make it responsive like the following, but it doesn't work at all (Bootstrap v5.0.2):

    <p class="fs-1 fs-md-3">Test text</p>

It seems that adding breakpoints to "fs" is not allowed, so "fs-md-3" cannot be recognized.  Could anyone tell me a method to display smaller font size on smaller screens? I don't need smooth transitions like using RFS. Thank you very much!

**Comment by ffoodd** [comment]
We're using RFS under the hood, with a minimum size of 20, which explains why smaller font sizes won't get smaller.

Font size utilities can be made responsive using our utilities API by setting the responsive parameter to true, but that requires to compile Sass on your behalf.

**Reply by squashking** [reply]
@ffoodd  Thanks, could you tell me what is "the responsive parameter"? I want to set it and have a try.

**Reply by ffoodd** [reply]
Sure, [this is documented in our Utilities API page](https://getbootstrap.com/docs/5.1/utilities/api/#enable-responsive).
